src/org/traccar/protocol/MiniFinderProtocolDecoder.java

```diff
@@ -34,44 +34,145 @@ public MiniFinderProtocolDecoder(MiniFinderProtocol protocol) {
         super(protocol);
     }
 
-    private static final Pattern PATTERN = new PatternBuilder()
-            .expression("![A-D],")
+    // The !A record is not in the documentation for MiniFinder or EV-07S.
+    // Eview, the manufacturer, indicated that resellers could implement new record types.
+    // Using the data in MiniFinderProtocolDecoderTest.java as a guide
+    private static final Pattern PATTERN_A = new PatternBuilder()
+            .expression("!A,")
             .number("(d+)/(d+)/(d+),")           // date
             .number("(d+):(d+):(d+),")           // time
             .number("(-?d+.d+),")                // latitude
             .number("(-?d+.d+),")                // longitude
-            .number("(d+.?d*),")                 // speed
+            .number("(d+.?d*),")                 // unknown
```

Comment:
I think instead of assuming that there will be 3 parameters, we should put `any()` here.